packit/config.py

```diff
@@ -333,6 +332,18 @@ def get_from_dict(cls, raw_dict: dict, validate=True) -> "PackageConfig":
         downstream_package_name = cls.get_deprecated_key(
             raw_dict, "downstream_package_name", "package_name"
         )
+        specfile_path = raw_dict.get("specfile_path", None)
+        if specfile_path:
+            specfile_path = str(Path(specfile_path).absolute().resolve())
```

Comment:
I don't see any `absolute()` in docs, but [resolve()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve) doc says: "Make the path absolute"

Reply:
nice catch!

at some point we could also start using Path internally instead of using str as we do now